app/src/main/java/com/noalecohen/dispatcher/view/activity/SearchActivity.kt

```diff
@@ -0,0 +1,105 @@
+package com.noalecohen.dispatcher.view.activity
+
+import android.content.Intent
+import android.os.Bundle
+import android.view.View
+import android.view.inputmethod.EditorInfo
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.noalecohen.dispatcher.adapter.ArticleAdapter
+import com.noalecohen.dispatcher.databinding.ActivitySearchBinding
+import com.noalecohen.dispatcher.decoration.TopSpacingItemDecoration
+import com.noalecohen.dispatcher.viewmodel.ArticlesViewModel
+import com.noalecohen.dispatcher.viewstate.RequestState
+
+
+class SearchActivity : AppCompatActivity() {
```

Comment:
Why did you implement this as an activity and not a fragment?

Reply:
Because.. I need to cover the entire screen, not just the FrameLayout of the tabs... 
So.. should I add another FrameLayout in MainActivity to cover the entire screen for loading the SearchFragment?